src/t8_geometry/t8_geometry_implementations/t8_geometry_examples.hxx

```diff
@@ -99,17 +99,50 @@ struct t8_geometry_triangulated_spherical_surface: public t8_geometry_with_verti
   }
 
   /**
-   * Map the faces of an oktaeder to a spherical surface.
+   * Map the faces of an octahedron/icasohedron to a spherical surface.
```

Comment:
Please do a search and replace for `icasohedron` and `icosahedron`
```suggestion
   * Map the faces of an octahedron/icosahedron to a spherical surface.
```

Reply:
Done.